Some extremely minor punctuation fixes

dericksonmark opened the pull request:
Some minor changes:
- I added a period between 'That's it' and 'That's the seed'
- I added a period at the end of 'can be found here' in the main paragraph

dericksonmark committed:
Add reproduction steps Pastebin link

In Sal's video description, there was a link to a Pastebin page that included full reproduction steps for pack.png.

Also, some minor changes:
- I added a period between 'That's it' and 'That's the seed'
- I added a period at the end of 'can be found here' in the main paragraph

laundmo commented:
I would love to merge but those reproduction steps, ugh. its way easier than using a nbt explorer, earthcomputer made a mod so you can input seeds in 1.2.2a... 

i will probably merge this and write my own reproduction steps instead as soon as i get around to actually writing that

dericksonmark commented:
Hello! Thanks for taking a look at the pull request. For now, I will edit it to be only the punctuation fixes. (sorry it took me so long to reply.)

dericksonmark committed:
Some extremely minor punctuation fixes

That's what I'll change the pull request title to. This commit only removes the reproduction steps that I added in a previous commit.

dericksonmark commented:
I'm closing this pull request, as the website no longer seems to exist.
